src/supermarket/app/models/cookbook_version.rb

```diff
@@ -11,7 +11,8 @@ class CookbookVersion < ApplicationRecord
   has_many :metric_results, -> { includes :quality_metric }, inverse_of: :cookbook_version
 
   belongs_to :cookbook
-  belongs_to :user
+  belongs_to :user, -> { includes :chef_account }, inverse_of: :cookbook_versions
```

Comment:
Have we defined the `inverse_of` in user model as well. Because this will be useful only when it's declared in a bidirectional manner i.e. both `has_many`/`has_one` and `belongs_to`